README.md

```diff
@@ -1 +1,61 @@
-# eoslime-shape-react
\ No newline at end of file
+# eoslime-shape-react
+
+The project is a complete and ready to use dApp with predefined React front-end and EOSLime integration. It consists of an EOS smart contract, deployment script, unit tests and a web part. The dApp represents a simple todo list, showing how to read and record a data on the blockchain through the browser. In gives a helpful example and foundation for a further development of more sophisticated dApp/s.
```

Comment:
The purpose of the project is to help you onboard on EOS easily by examples.

Project includes

- smart contract
- deployment script
- unit tests
- web interface

Reply:
updated